### DISCUSSION TITLE
Digital/2024/lab3/tasks/

### DISCUSSION THREAD

**Discussion by giscus** [wintermelon008/USTC-DigitalLab-Discussion#39] [Announcements]
# Digital/2024/lab3/tasks/

https://soc.ustc.edu.cn/Digital/2024/lab3/tasks/

**Comment by needs-money** [comment]
青春无极限，统一冰红茶ᕙ(`▿´)ᕗ

**Comment by SBS-19** [comment]
😘😘

**Comment by EthanCu1** [comment]
七段数码管我有一个问题，我希望复位时8个数码管全部恢复成最初熄灭的状态，于是我尝试性地复位时给seg_data设为4‘bz，seg_an为 3’b000 ， 实际情况时，其余7个位置数码管无任何显示，然而 0 位置的数码管会显示为F ， 请问怎样地编码可以让复位时最后一个数码管也完全熄灭呢 ？

**Reply by EthanCu1** [reply]
做到选必做的题目看到了，原来就是要有个位置常亮
